browser/src/control/jsdialog/Util.KeyboardGridNavigation.ts

```diff
@@ -9,10 +9,6 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-/*
- * JSDialog.ScrollableBar - helper for creating toolbars with scrolling left/right
- */
```

Comment:
it would be nice to have this comment but the good description, so people who see it first time will know what it does (in human readable language :) )